samples/exceptions/ExceptionMethod.java

```diff
@@ -0,0 +1,31 @@
+package exceptions;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class ExceptionMethod {
+
+    public static void main(String[] args) {
```

Comment:
Why adding another main() method? We have already one in Main class. So to have clean app architecture, we should run all examples from `ExceptionSamples` class.